bindings/python/crocoddyl/multibody/contacts/contact-2d.cpp

```diff
@@ -43,6 +43,7 @@ void exposeContact2D() {
            "Compute the derivatives of the 2D contact holonomic constraint.\n\n"
            "The rigid contact model throught acceleration-base holonomic constraint\n"
            "of the contact frame placement.\n"
+           "It assumes that that calc has been run first.\n"
```

Comment:
It is present in multiple documentation changes that you did. The above command should take care of all of them